arangod/Graph/PathManagement/PathValidator.h

```diff
@@ -126,6 +134,10 @@ class PathValidator {
 
   auto evaluateVertexExpression(arangodb::aql::Expression* expression,
                                 arangodb::velocypack::Slice value) -> bool;
+
+  auto checkValidDisjointPath(typename PathStore::Step const& lastStep,
+                              bool isSatelliteLeader)
```

Comment:
The isSatelliteLeader should be part of the Options

Reply:
Done